src/files/domain/models/FileDownloadSizeMode.ts

```diff
@@ -0,0 +1,5 @@
+export enum FileDownloadSizeMode {
+  ALL = 'All',
+  ORIGINAL = 'Original',
+  ARCHIVAL = 'Archival',
```

Comment:
Just a quick question, so archival is with the tabular files processed, original is with the tabular files not processed and ALL I assume it means both ORIGINAL and ARCHIVAL tabular files, is it?

Reply:
That's it, as you describe!